reactor-netty-http/build.gradle

```
@@ -151,6 +151,15 @@ dependencies {
 	testRuntimeOnly "org.slf4j:jcl-over-slf4j:$slf4jVersion"
 	testRuntimeOnly "ch.qos.logback:logback-classic:$logbackVersion"
 
+	// Needed for Brotli compression
+	testImplementation "com.aayushatharva.brotli4j:brotli4j:$brotli4jVersion"
+	if (osdetector.classifier in ["linux-aarch_64"] || ["osx-aarch_64"]) {
```

I think we need this change:

```suggestion
	if (osdetector.classifier == "linux-aarch_64" || osdetector.classifier == "osx-aarch_64") {
```

---

the classifiers for the brotli natives are interesting decision